applications/configuration.md

```diff
@@ -0,0 +1,103 @@
+# Configuration
+
+When setting up configurables for an application we need to take into account several things.
+
+- Private and sensitive variables
+  - These can be stored in a private repo in GitLab
```

Comment:
For particularly sensitive variables (such as credentials to other services), we may prefer to store these in RatticDB rather than a private repo?